### ISSUE TITLE
Xiaomi Pedestal Fan (zhimi-fan-sa1) discover issue

### ISSUE THREAD

**Issue by SaiT2** [rytilahti/python-miio#681]
Hi guys,

I'm triyng to control my Xiaomi Pedestal Fan through `miiocli fansa1` and i get a strange behaviour:

When the device isn't connected to my local network, i get the possibility to connect to the network of the fan (ssid: zhimi-fan-sa1). Once connected to the fan, i get the token through `miio discover` and i use that with `miiocli` command line tool to control the device. So, 
`miiocli fansa1 --ip FAN_IP --token FAN_TOKEN on` 
and all other functions works (like off, set_angle, ecc..) 
When i connect the fan to the local network (either through the Mi Home application and through the `miio configure` with --ssid and --passwd options set) the `miio discover` on my local network does not detect the device, although i can see through the router of my local network that it's connected and it has an assigned IP. Consequently
`miiocli fansa1 --ip LAN_IP --token FAN_TOKEN on` 
does not work.
Can you help me solve this? Thank you

**Comment by rytilahti** [comment]
The provisioning will regenerate a new token, so you need to find a way (some examples at https://www.home-assistant.io/integrations/vacuum.xiaomi_miio/#retrieving-the-access-token) to obtain the token after you have configured the device to join the network.

**Comment by SaiT2** [comment]
Thanks! but is this a specific behaviour of the this fan device? because i follow the procedure above for other device that mantains the same token during this procedure (in particular  zimi.powerstrip.v2 )
I will try another way to get the token! Thank you so much

**Comment by rytilahti** [comment]
Oh, that may be different on different devices, but in most of the cases I have heard about it gets reseted (I suppose the functionality is directly built-in to the miio sdk for device vendors).

I don't understand your second question, is the discovery working or not?

**Comment by SaiT2** [comment]
I'm sorry, my explanation wasn't very clear.

When i connect the fan to the local network (with Mi Home for example) the fan is not discovered.
When i use `miio discover` on my local network it discovers many device, some of them with token and some of them without token. So, regardless the token, the fan is not discovered, but it is connected (work through Mi Home and have its own IP in the network)

**Comment by rytilahti** [comment]
Oh, got it! So it's not responsive at all after the initial provisioning, I have not yet heard about such behavior earlier. Unfortunately there is not much we can do about it..

**Comment by SaiT2** [comment]
Yes, i confirm that when the fan is not yet connected and has its own network, if i connect to its network i can use `miio discover` and i can communicate with it, but when the fan go under the local network it is no longer discovered. Let me know if i can give other detailed information
Thanks!
